pkgs/applications/emulators/ukmm/default.nix

```diff
@@ -0,0 +1,60 @@
+{ lib
+, atk
+, glib
+, cmake
+, gtk3-x11
+, pkg-config
+, rustPlatform
+, wrapGAppsHook
+, fetchFromGitHub
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "ukmm";
+  version = "0.7.1";
+
+  src = fetchFromGitHub {
+    owner = "NiceneNerd";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "sha256-TvRaaAVJPqvTTqs+d5d4y1KmDTvE41uP5AykLtHxD9w=";
+  };
+
+  cargoLock = {
+      lockFile = ./Cargo.lock;
+      outputHashes = {
+        "catppuccin-egui-1.0.2" = "sha256-+ILfvDgZxe/QPJuVqIbRjaHNovpRAX+ym2QZ96glb4w=";
+        "ecolor-0.20.0" = "sha256-uTDkNRWsA1nM8Qhb0X2LjVDRuaW31vWxR8kDLL27BVE=";
+        "egui-notify-0.4.0" = "sha256-jybtUnv9xqzulZ5nfg+T1u8iTOsPjKGVVQ7JhwbvPdU=";
+        "egui_commonmark-0.6.0" = "sha256-hsVbtL2F+jifnzN6FgcDAVtLd1bVxTs0twn0SMvq9eU=";
+        "egui_dock-0.2.1" = "sha256-gGIO0boXKxLu0ABDH/uJhEZEoE/ql8E65LRmr0Xhv3s=";
+        "junction-0.2.0" = "sha256-6+pPp5wG1NoIj16Z+OvO4Pvy0jnQibn/A9cTaHAEVq4=";
+        "msbt-0.1.1" = "sha256-PtBs60xgYrwS7yPnRzXpExwYUD3azIaqObRnnJEL5dE=";
+        "msyt-1.2.1" = "sha256-aw5whCoQBhO0u9Fx2rTO1sRuPdGnAAlmPWv5q8CbQcI=";
+      };
+  };
+
+  doCheck = false;
+  RUSTC_BOOTSTRAP = true;
+
+  nativeBuildInputs = [
+    cmake
+    pkg-config
+    wrapGAppsHook
+  ];
+
+  buildInputs = [
+    atk
+    glib
+    gtk3-x11
+  ];
+
+  meta = with lib; {
+    description = "A new mod manager for The Legend of Zelda: Breath of the Wild. Successor to BCML.";
+    homepage = "https://github.com/NiceneNerd/ukmm";
+    changelog = "https://github.com/NiceneNerd/ukmm/releases/tag/v${version}";
+    license = licenses.gpl3Plus;
+    maintainers = with maintainers; [];
```

Comment:
This shouldn't be empty (See [CONTRIBUTING.md#submitting-changes](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#submitting-changes)), would you be willing to maintain this?

You could add yourself to [the maintainers list](https://github.com/NixOS/nixpkgs/blob/f17aa727523ece00fcadf56f49340bcdd81a74c1/maintainers/maintainer-list.nix).

Reply:
I haven't played any Zelda in my life so I don't think I want to pay attention to this. Also, I don't know nix language. Just borrowing codes from others and piercing puzzles together enough to get this working.

Reply:
Ok - I'd be happy to maintain it then!